experimenter/experiments/straight.py

```diff
@@ -54,15 +56,16 @@ def generate_pipeline(self, preprocessor: PrimitiveBase, classifier: PrimitiveBa
         """
 
         # Creating Pipeline
-        pipeline_description = EZPipeline(context=Context.TESTING)
+        architecture = PipelineArchDesc("straight")
+        pipeline_description = EZPipeline(arch_desc=architecture, context=Context.TESTING)
 
         add_initial_steps(pipeline_description)
         preprocessor_used = False
 
         # Preprocessor Step
         if preprocessor:
             preprocessor_step = PrimitiveStep(primitive_description=preprocessor.metadata.query())
-            req_args = get_required_args(pre)
```

Comment:
This seems to have been a bug that sneaked in. There doesn't seem to be a `pre` variable anywhere in scope.